Core/src/MagneticField/BFieldMapUtils.cpp

```diff
@@ -27,6 +27,28 @@
 using Acts::VectorHelpers::perp;
 using Acts::VectorHelpers::phi;
 
+namespace {
+auto getMinMaxAndBinCount(std::vector<double>& xPos) {
```

Comment:
can this be deduplicated with the function below also named `getMinMaxAndBinCount`?

Reply:
How about adding it to `Helpers.hpp`?
https://github.com/acts-project/acts/blob/main/Core/include/Acts/Utilities/Helpers.hpp

Reply:
seems a bit generic but works for me 👍